pom.xml

```diff
@@ -642,17 +642,16 @@
                 <plugin>
                     <groupId>org.glassfish.copyright</groupId>
                     <artifactId>glassfish-copyright-maven-plugin</artifactId>
-                    <version>2.0</version>
                     <configuration>
                         <excludeFile>etc/config/copyright-exclude</excludeFile>
                         <!--svn|mercurial|git - defaults to svn-->
                         <scm>git</scm>
                         <!-- turn on/off debugging -->
-                        <debug>false</debug>
+                        <debug>true</debug>
```

Comment:
why in debug mode?

Reply:
I have added it temporarily to know why is it failing in Travis. Currently it fails but there is no information of the error.

Reply:
for now build did not even started since the last change :) Travis seems to be really slow recently.
Regarding additional info - it might not show it because of reduced output in Travis. We had to reduce it due to Travis' limitations, otherwise all build threads would fail. May be you can temporary try to run it with full log output (just for copyright-check thread) and this will show what actually happened there?

Reply:
ah, no, copyright already runs with full output, so may be -e key will help when build finally starts :)

Reply:
I tried it yesterday with -e already but the stacktrace doesn't help, you can see it here:
https://travis-ci.org/github/eclipse-ee4j/jersey/jobs/747370995

Hopefully debug=true will help.

Locally there are no problems. I don't really understand why in Travis it works different.

Reply:
the issue with no output on failures seems to be caused by this hard-coded logic inside the plugin:
https://github.com/eclipse-ee4j/glassfish-copyright-plugin/blob/94c7ba4c6d3151b57f92cf030f77661f1ff15d17/src/main/java/org/glassfish/copyright/CheckCopyrightMojo.java#L33
It's being always quiet in other words. And when you remove this hard-coded line (default is false), you can see why build fails. However fix requires new PR and then new plugin version shall be released...